### ISSUE TITLE
why not use webfinger.js?

### ISSUE THREAD

**Issue by silverbucket** [michielbdejong/ludbud#7]
I think it's a good idea to re-use as many existing, single purpose, libraries as possible. Like you are discussing in the remotestorage.js issues at the moment. So why don't we use https://github.com/silverbucket/webfinger.js here?

**Comment by michielbdejong** [comment]
Yeah, I originally wanted to reuse all existing rs.js code, but didn't manage to untangle it.

But yeah, I could replace [these 7 lines](https://github.com/michielbdejong/ludbud/blob/master/src/oauth.js#L60-L66) with webfinger.js, let me try that.

**Comment by silverbucket** [comment]
It would replace more than just those 7, but yeah - I think it's a good idea to use a bunch of small single purpose libraries which focus on their problem domain, hopefully with tests etc. so we can offload as much technical debt as possible. once you get together a build process and a package.json, you can pin the version to follow the 2.x branch, I'm following semver so the API wont break for 2.x

**Comment by michielbdejong** [comment]
OK, so I did this (took me exactly 30 minutes, looking at the comment dates :) ): https://github.com/michielbdejong/ludbud/pull/8
